File: PersonalAssistant.DataAccess/Entities/Conversation.cs
using PersonalAssistant.DataAccess.Identity;

namespace PersonalAssistant.DataAccess.Entities
{
    public class Conversation
    {
        public string Name { get; set; }
        public string Icon { get; set; }
        public string ConversationType { get; set; }        
        public int? RoomId { get; set; }
        public Room Room { get; set; }
        public int? UserId { get; set; }
        public User User { get; set; }
        public Message LastMessage { get; set; }
    }
}

File: PersonalAssistantUI/src/app/models/conversation/conversation.ts
import { Message, Room, User } from "..";
import * as Constants from "../../utils/constants";

export class Conversation {
    public name: string;
    public icon: number;
    public conversationType: string;
    public lastMessage: Message;
    public roomId: number;
    public room: Room;
    public userId: number;
    public user: User;    
    public unreadMessageCount: number;    
}